tutorials/build_system/gradle_guide.md

```diff
@@ -0,0 +1,225 @@
+That documentation explains three different ways to use Gradle for plugin uploads without exposing account credentials.
+
+
+## Common IntelliJ IDEA Gradle Plugin Configurations
+Combinations of attributes in different Gradle plugin tasks are often needed to create the desired build or IDE Development Instance environment.
+This section review some of the more common configurations.
+
+### Plugins Targeting IntelliJ IDEA
+IntelliJ Platform plugins targeting IntelliJ IDEA have the most straightforward Gradle plugin configuration.
+* Determine the version of IntelliJ IDEA to target; this is the version of the IntelliJ Platform.
+  This can be EAP (default) or determined from the [build number ranges](/basics/getting_started/build_number_ranges.md). 
+  * If a production version of IntelliJ IDEA is the desired target, set the `intellij` [version attributes](#intellij-platform-version) accordingly. 
+  * Set the necessary [plugin dependencies](#plugin-dependencies), if any. 
+* If the plugin project should be run or debugged in an IDE Development Instance based on the same IntelliJ IDEA version, no attributes need to be set. 
+  This is the default behavior and is the most common use case. 
+  * If the plugin project should be run or debugged in an IDE Development Instance based on a different IntelliJ IDEA version, set [the path](#running-against-alternate-versions-of-the-intellij-platform) to the desired version. 
+  * If the plugin project should be run using a JetBrains Java runtime other than the default for the IDE Development Instance, specify the [JetBrains Java runtime version](#running-against-alternate-versions-of-the-java-runtime). 
+
+### Plugins Targeting Other IntelliJ Platform-Based IDEs
+The Gradle plugin can also be configured for developing plugins to run in IDEs that are based on the IntelliJ Platform but are not IntelliJ IDEA.
+This section uses the example of developing a plugin for PhpStorm.
+It will be helpful to review the [PhpStorm Plugin Development](/products/phpstorm/phpstorm.md) section.
+
+* The Gradle plugin attributes describing the configuration of the IntelliJ Platform used to build the plugin project must be explicitly set. 
+  The type will be "IC" because the IntelliJ Platform is defined by the IntelliJ IDEA Community Edition.
+  The BRANCH.BUILD number of the IntelliJ Platform (IntelliJ IDEA CE) is the same as for the PhpStorm target.
+  Although the FIX (tertiary) number may differ between the same versions of the applications, it won't affect the match of the IntelliJ Platform.
+* All PhpStorm plugin projects have a dependency on the PhpStorm OpenAPI Library.
+  Any plugin targeting PhpStorm must list a dependency on the PHP plugin, and its version must be compatible with the target version of PhpStorm.
+  The plugin dependency must be declared using the Gradle plugin `intellij.plugins` attribute, which lists the FQN and version of the plugin dependency.
```

Comment:
there is no "FQN" but only `id` of a plugin

Reply:
Wishful thinking on my part.